lib/organization.js

```diff
@@ -273,4 +271,97 @@ export class Organization {
     // NB: this will not work for proposals using votingMachine's that are not the default one
     return this.votingMachine.vote(proposalId, choice, params);
   }
+
+  /**
+   * Returns global constraints currently registered into this DAO, as Array<OrganizationGlobalConstraintInfo>
+   * @param name like "TokenCapGC"
+   */
+  async globalConstraints(name) {
+    // return the global constraints registered on this controller satisfying the contract spec
+    // return all global constraints if name is not given
+    const constraints = await this._getConstraints();
+    if (name) {
+      return constraints.filter(s => s.name === name);
+    } else {
+      return constraints;
+    }
+  }
+
+  /**
+   * returns global constraints currently in this Organization, as OrganizationGlobalConstraintInfo
+   */
+  async _getConstraints() {
+    // private method returns all registered schemes.
```

Comment:
constraints not schemes, also why do we need this comment alongside the block comment above the function?

Reply:
yeah, there are a lot of old comments in this code from long before my time that I tend to just block out of my mind...

Reply:
deleted